quelpa.el

```diff
@@ -1689,21 +1721,25 @@ If t, `quelpa' tries to do an upgrade.
 
 :stable
 
-If t, `quelpa' tries building the stable version of a package."
+If t, `quelpa' tries building the stable version of a package.
+
+:force
+
+If t, `quelpa' tries to do an upgrade regardless state of local package repo."
   (while plist
     (let ((key (car plist))
           (value (cadr plist)))
       (pcase key
         (:upgrade (setq quelpa-upgrade-p value))
-        (:stable (setq quelpa-stable-p value))))
+        (:stable (setq quelpa-stable-p value))
+        (:force (setq quelpa-force-upgrade-p value))))
     (setq plist (cddr plist))))
 
 (defun quelpa-package-install-file (file)
   "Workaround problem with `package-install-file'.
 `package-install-file' uses `insert-file-contents-literally'
-which causes problems when the file inserted has crlf line
-endings (Windows). So here we replace that with
-`insert-file-contents' for non-tar files."
+which causes problems when the FILE inserted has crlf line endings (Windows).
```

Comment:
I'm not sure, but this may need wrapping to avoid `checkdoc` warnings (although we may want to do a separate `checkdoc` pass on all docstrings in Quelpa's code later).

Reply:
This is 76 characters, just make it under 80 :). I agree we should do separate checkdoc later